benchmarks/benchmarks/utils.py

```diff
@@ -1,19 +1,30 @@
 from __future__ import annotations
 
+import warnings
 from functools import cache
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Literal
 
+import numpy as np
 import pooch
+from anndata import concat
 
 import scanpy as sc
 
 if TYPE_CHECKING:
     from anndata import AnnData
 
+    Dataset = Literal["pbmc68k_reduced", "pbmc3k", "bmmc", "lung93k"]
+
 
 @cache
 def _pbmc68k_reduced() -> AnnData:
-    return sc.datasets.pbmc68k_reduced()
+    adata = sc.datasets.pbmc68k_reduced()
+    mapper = dict(
+        percent_mito="pct_counts_mt",
+        n_counts="total_counts",
+    )
```

Comment:
In general, since I saw you did this elsewhere, why do you use `dict`?  I believe it is much slower, but even so, I find it harder to read.

Reply:
if the speed of `dict` is noticable, one has bigger problems. I use it whenever there are `snake_case` keys, i.e. when one is creating a kwargs dict. Use kwargs to define kwargs.